frontend/src/main/java/com/psephoi/mvc/controller/PsephoiRestController.java

```diff
@@ -0,0 +1,79 @@
+package com.psephoi.mvc.controller;
+ 
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.util.UriComponentsBuilder;
+import org.springframework.web.bind.annotation.CrossOrigin;
+
+
+import com.psephoi.mvc.model.Poll;
+import com.psephoi.mvc.manager.PollManager;
+ 
+@RestController
+@CrossOrigin(origins= "*", allowedHeaders = "*")

```

Comment:
I don't think you need any of these CrossOrigin annotations.  The only time I've ever seen CrossOrgin annotations used in a login/SSO authentication/authorization endpoints.